api/models/Token.js

```diff
@@ -0,0 +1,11 @@
+module.exports = bookshelf.Model.extend({
+  tableName: 'tokens',
```

Comment:
instead of creating a new model, we could also provide the same functionality by using the existing `LinkedAccount` model, setting its `provider_key` value to 'token' and `provider_user_id` to the token's value.